google/cloud/bigquery/table.py

```diff
@@ -293,15 +293,18 @@ class Table(object):
     """
 
     _PROPERTY_TO_API_FIELD = {
-        "friendly_name": "friendlyName",
+        "encryption_configuration": "encryptionConfiguration",
         "expires": "expirationTime",
-        "time_partitioning": "timePartitioning",
-        "partitioning_type": "timePartitioning",
+        "external_data_configuration": "externalDataConfiguration",
+        "friendly_name": "friendlyName",
+        "mv_enable_refresh": "materializedView",
```

Comment:
I wonder if we can use a better prefix?  For logical views we're already using `view_` prefix, maybe mview or matview? My worry that mv is too terse here and/or people will overlook them in favor of the logical properties.